code/_globalvars/lists/objects.dm

```diff
@@ -38,4 +38,6 @@ GLOBAL_LIST_EMPTY(wire_name_directory)
 GLOBAL_LIST_EMPTY(ai_status_displays)
 
 GLOBAL_LIST_EMPTY(mob_spawners) 		    // All mob_spawn objects
+GLOBAL_LIST_EMPTY(latejoin_mob_spawners)	// All mob_spawn objects that can be viewed and accessed from the lobby.

```

Comment:
Wouldn't it be far better to have the spawn menu check each mob spawner individually for latejoin visibility? The method of storing latejoin visible mob spawners in a separate list is a little messy and limits adminbus

Reply:
I'll see what can be done. While I wanted to avoid the lobby join menu from having to run such check function for each ghost role spawner, I was pretty very tired when I coded this yesterday.